.github/workflows/release.yaml

```diff
@@ -112,10 +114,29 @@ jobs:
           echo "::notice::python version: $(python --version)"
           echo "::notice::rustup version: $(rustup show)"
           echo "::notice::wasm-pack version: $(wasm-pack --version)"
+          echo "::notice::OpenSSL version:" $(openssl version)
 
-      - name: Install native packages (Ubuntu only)
+      - name: Install native packages and OpenSSL 1.1 (Ubuntu only)
```

Comment:
It would be good to comment that this is a workaround, and it should be removed when we no longer depend on the deprecated OpenSSL 1.1.